monkey/infection_monkey/config.py

```diff
@@ -272,5 +281,17 @@ def get_exploit_user_password_or_hash_product(self):
     PBA_linux_filename = None
     PBA_windows_filename = None
 
+    @staticmethod
+    def hash_sensitive_data(sensitive_data):
+        """
+        Hash sensitive data (e.g. passwords). Used so the log won't contain sensitive data plain-text, as the log is
+        saved on client machines plain-text.
+
+        :param sensitive_data: the data to hash.
+        :return: the hashed data.
+        """
+        password_hashed = hashlib.sha512(sensitive_data).hexdigest()
+        return password_hashed
```

Comment:
I don't think that's very important, but, if we'll only use it for logging(based on comment), then maybe should just include "(SHA512):  " in the returned string.

Reply:
Prefer to do in the format string and not in the data